index.html

```diff
@@ -1,17 +1,20 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
     <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link href="style.css" rel="stylesheet" />
+        <title>Microverse Second Trial</title>
     </head>
     <body>
-        <div id="main">
+        <main>
             <h1 id="title">Dr. Norman Borlaug</h1>
             <h4 id="sub-title">The man who saved a billion lives</h4>
-            <div id="img-div">
-                <img src="image.jpg" width="100%" height="auto" style="align-content: center;"><br><br>
+            <section id="img-div">
+                <img src="image.jpg" alt="main-image"><br><br>
                 <span id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</span>
-            </div>
-            <div id="tribute-info">
+            </section>
+            <section id="tribute-info">
```

Comment:
* Hi, @ShahierNashaat  this is an example of how to use inline comments 
* Click on the File changed the link and select the block of code where you want to request a change on and comment below the code ( ie see how i commented under the code on line 18, i can now make my review saying " Kindly use classes instead of id's, this is the best practice as it enables you to use the styling in multiple places  ")